scripts/firezone-client-gui-install.sh

```diff
@@ -9,20 +9,13 @@ set -euox pipefail
 # `apt-get` needs either a leading `./` or `/` to recognize a local file path
 DEB_PATH=$(realpath "$1")
 GROUP_NAME="firezone-client"
-SERVICE_NAME="firezone-client-ipc"
 
 echo "Installing Firezone..."
 sudo apt-get install --yes "$DEB_PATH"
 
 echo "Adding your user to the $GROUP_NAME group..."
-# Creates the system group `firezone-client`
-sudo systemd-sysusers
 sudo adduser "$USER" "$GROUP_NAME"
```

Comment:
Unfortunately I can't get rid of this, even Docker has the manual group add step: https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user

I checked the default groups that desktop users have in Ubuntu and none of them seemed appropriate except maybe `sudo`, but that doesn't allow unprivileged users to run the GUI.

There's a few obstacles to fully automating it:
- dpkg runs as root, so we may not know the username, or we'd have to look in `$SUDO_USER` and hope it matches
- An admin may be installing it for someone else
- An MDM may be installing it for someone else
- Debian packages just aren't supposed to make changes like this to user accounts

Reply:
Makes sense.